src/arrays/ArrayUtil.java

```diff
@@ -1,61 +1,99 @@
 package arrays;
 
 
+import java.util.Arrays;
+
 /**
  * An utility class containing methods for convenient work with arrays
  */
 public class ArrayUtil {
 
-
     /**
      * Create an array with the given size, fill it with the different values and print it.
      */
     public static void createArray(int size) {
-
+        int[] array = new int[size];
+        for (int i = 0; i < array.length; i++) {
+            array[i] = (i + 1) * 2;
+        }
+        System.out.println("Array is created " + Arrays.toString(array));
     }
 
     /**
      * Create two dimensional array with the given sizes, fill it with different values and print it.
      */
     public static void create2dArray(int x, int y) {
+        int[][] twoDArray = new int[x][y];
+        for (int i = 0; i < twoDArray.length; i++) {
+            for (int j = 0; j < twoDArray[i].length; j++) {
+                twoDArray[i][j] = (i + 1) * (j + 1);
+            }
+        }
+        System.out.println("TwoD array is created " + Arrays.deepToString(twoDArray));
     }
 
+
     /**
      * Add the given value at the beginning of the given array
      *
      * @return new array.
      */
     public static int[] addFirst(int[] array, int valueToAdd) {
 
-        return null;
+        int[] result = new int[array.length + 1];
+        System.arraycopy(array,0, result, 1, array.length);
+        result[0] = valueToAdd;
+        array = result;
+        return array;
     }
 
+
     /**
      * Implement a method that will check if the given value contains in the array
      *
      * @return true if the array contains the given value, otherwise return false.
      */
     public static boolean contains(int[] array, int value) {
-
+        for (int i = 0; i < array.length; i++) {
+            if (array[i] == value) {
+                return true;
+            }
+        }
         return false;
     }
 
+
     /**
      * @param array An array to search in.
      * @param value The value to search for.
      * @return the first index of value in the array, if array does not contain given element return -1.
      */
     public static int indexOf(int[] array, int value) {
-
-        return 0;
+        for (int i = 0; i < array.length; i++) {
+            if (array[i] == value)
+                return i;
+        }
+        return -1;
     }
 
+
     /**
      * Remove the element by the given index.
      * Print new array
      */
     public static void remove(int[] array, int index) {
-
+        if (index < 0)
```

Comment:
No need to continue invocation this method if **index < 0 OR index > array.length - 1**, just put **return;** after **System.out.println("Incorrect index");**